src/metabase/util.clj

```diff
@@ -74,6 +60,20 @@
   [^CharSequence s]
   (.. s toString (toUpperCase (Locale/US))))
 
+(defn screaming-snake-case
+  "Turns `strings-that-look-like-deafening-vipers` into `STRINGS_THAT_LOOK_LIKE_DEAFENING_VIPERS`."
+  [s]
+  (upper-case-en (str/replace s "-" "_")))
+
+(defn capitalize-first-char
+  "Like string/capitalize, only it ignores the rest of the string
+  to retain case-sensitive capitalization, e.g., PostgreSQL."
+  [s]
+  (if (< (count s) 2)
+    (upper-case-en s)
+    (str (upper-case-en (subs s 0 1))
```

Comment:
This is a very generic function, I'm not sure this is the right call in every case.

Reply:
When might we want to use `str/upper-case` instead of `upper-case-en` here?